README.md

```diff
@@ -199,33 +199,37 @@ The base path to which all `template` paths are relative.
 
 Type: `String`
 Optional: `true`
-Default: `main.hbs` (provided by Sugarcoat).
+Default: `main.hbs` (provided by Sugarcoat)
+Relative: `template.cwd`
 
-Path (relative to `template.cwd`, if provided) to the Handlebars layout that will define the layout of the site.
+Path to the Handlebars layout that will define the layout of the site.
 
 ### `template.partials` ###
 
 Type: [Standardized File Format](#standardized-file-format)
 Optional: `true`
 Default: See [templating](#templating) for a list of Sugarcoat's provided partials.
+Relative: `template.cwd`
 
-Partial file(s) (relative to `template.cwd`) to register with Handlebars. If any partials use a [reserved name](#reserved-partial-names), the respective partial will override the one provided by Sugarcoat. If you choose to include an object or an array of objects, you must include a `src` and `options`. If you do not choose to include options through an object, Sugarcoat will default it's glob options to `nodir: true`.
+Partial file(s) to register with Handlebars. If any partials use a [reserved name](#reserved-partial-names), the respective partial will override the one provided by Sugarcoat. If you choose to include an object or an array of objects, you must include a `src` and `options`. If you do not choose to include options through an object, Sugarcoat will default it's glob options to `nodir: true`.
```

Comment:
The lines "If you choose to include an object or an array of objects, you must include a `src` and `options`. If you do not choose to include options through an object, Sugarcoat will default it's glob options to `nodir: true`." are still confusing to me. You are saying that if you include an object or an array of object you *have* to include `src` and `options` but then you say in the next line that you *don't* have to include `options`.